### ISSUE TITLE
Update Korean translations

### ISSUE THREAD

**Issue by minjikarin** [squidfunk/mkdocs-material#5172]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "ko",
  "action.edit": "이 페이지를 편집",
  "action.skip": "콘텐츠로 이동",
  "action.view": "페이지소스 보기",
  "announce.dismiss": "다시 안보기",
  "blog.archive": "아카이브",
  "blog.categories": "카테고리",
  "blog.categories.in": "카테고리",
  "blog.continue": "계속 읽기",
  "blog.draft": "임시 저장",
  "blog.index": "Index로 돌아가기",
  "blog.meta": "메타데이터",
  "blog.references": "관련 링크",
  "clipboard.copy": "클립보드로 복사",
  "clipboard.copied": "클립보드에 복사됨",
  "consent.accept": "동의 허락",
  "consent.manage": "동의 허락 관리",
  "consent.reject": "동의 거부",
  "footer": "하단/푸터",
  "footer.next": "다음",
  "footer.previous": "이전",
  "header": "상단/헤더",
  "meta.comments": "댓글",
  "meta.source": "출처",
  "nav": "네비게이션",
  "readtime.one": "읽는시간 1분",
  "readtime.other": "읽는시간 #분",
  "rss.created": "RSS 피드 생성완료",
  "rss.updated": "RSS 피드 업데이트완료",
  "search": "검색",
  "search.placeholder": "검색",
  "search.share": "공유",
  "search.reset": "지우기",
  "search.result.initializer": "검색 초기화",
  "search.result.placeholder": "검색어를 입력하세요",
  "search.result.none": "검색어와 일치하는 문서가 없습니다",
  "search.result.one": "1개의 일치하는 문서",
  "search.result.other": "#개의 일치하는 문서",
  "search.result.more.one": "이 문서에서 1개의 검색 결과 더 보기",
  "search.result.more.other": "이 문서에서 #개의 검색 결과 더 보기",
  "search.result.term.missing": "포함되지 않은 검색어",
  "select.language": "언어설정",
  "select.version": "버전 선택",
  "source": "저장소로 이동",
  "source.file.contributors": "참여자들",
  "source.file.date.created": "작성일",
  "source.file.date.updated": "마지막 업데이트",
  "tabs": "탭",
  "toc": "목차",
  "top": "맨위로"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by minjikarin** [comment]
completed the rest of translation for korean

**Comment by squidfunk** [comment]
Perfect, thanks! Added in 6538b35a3.

**Comment by squidfunk** [comment]
Released as part of 9.1.2.
